src/controller/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
```

Comment:
In the 'get' function, you are sending 'users' to the client before sending a 404 status code. If 'users' is undefined or null, you should send the 404 status code first.